### PR TITLE
Action my proposals

### PR DESCRIPTION
**Pull request by alexisljn**
Creation of the ActiveDataProvider in charge to request
the pending proposals of a member.

### CODE REVIEW

File: controllers/ProposalController.php [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+        return $myPendingProposals;
+    }
+
+    /**
+     * Returns the reviewed proposals of a member.
+     *
+     * @return ActiveDataProvider
+     */
+    private function getMyReviewedProposals(): ActiveDataProvider
```
Comment (EmileCalixte) [inline controllers/ProposalController.php]: The name of this function is ambiguous for me: the name suggests that the function returns proposals that have at least one review, while it returns the proposals that are NOT pending. A proposal can be reviewed by someone but still pending.
I suggest `getMyProposalsNotPending`.

File: controllers/ProposalController.php [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+     * @return string
+     */
+    public function actionMyProposals(): string
+    {
+        $myPendingProposals = $this->getMyPendingProposals();
+        $myReviewedProposals = $this->getMyReviewedProposals();
+
+        return $this->render('my-proposals',[
+            'myPendingProposals' => $myPendingProposals,
+            'myReviewedProposals' => $myReviewedProposals
```
Comment (EmileCalixte) [inline controllers/ProposalController.php]: This name is ambiguous, see my comment line 68

File: views/proposal/my-proposals.php [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+            'value' => function($proposal)
+            {
+                /** @var \app\models\databaseModels\Proposal $proposal */
+
+                return $proposal->getReviews()->where(['status' => 'rejected'])->count();
+            }
+        ]
+    ]
+]); ?>
+<strong>My reviewed proposals</strong>
```
Comment (EmileCalixte) [inline views/proposal/my-proposals.php]: This name is ambiguous, see my comment in ProposalController.php line 68.
I suggest 'Proposals history' or something like that.

**Review by EmileCalixte — COMMENTED** [review]
'myReviewedProposals' is ambiguous, see comments for details.

File: controllers/ProposalController.php (hunk 1 above)
Comment (alexisljn) [reply controllers/ProposalController.php]: Agreed, I'll change that this evening

File: views/proposal/my-proposals.php (hunk 3 above)
Comment (alexisljn) [reply views/proposal/my-proposals.php]: Proposals history sounds good

**Review by EmileCalixte — APPROVED** [review]
Perfect

**Pull request merged**